fsspec/implementations/tests/test_memory.py

```diff
@@ -23,19 +23,6 @@ def test_strip(m):
     assert m._strip_protocol("/b/c/") == "/b/c"
 
 
-def test_put_single(m, tmpdir):
```

Comment:
This test is no longer correct, but rather than change the test in line with current functionality I have deleted it as it is equivalent to the "copy file to directory" test.